scripts/simple-scylla-cluster-manager.py

```diff
@@ -0,0 +1,776 @@
+                node from the cluster, this is left to the user.
+                """),
+            help="stop the node if running and remove all its data (doesn\'t remove the node from the cluster)")
+
+    parser_status = subparsers.add_parser(
+            "status",
+            formatter_class=RawDescriptionDefaultsHelpFormatter,
+            description=textwrap.dedent(f"""\
+                Print the status of all the nodes in the cluster
+
+                Example:
+                $ ./{app_name} status
+                [
+                    {{
+                        "index": 1,
+                        "ip": "127.0.150.1",
+                        "status": "UP",
+                        "pid": 254315
+                    }},
+                    {{
+                        "index": 2,
+                        "ip": "127.0.150.2",
+                        "status": "UP",
+                        "pid": 254323
+                    }}
+                ]
+
+                Where:
+                * index - the index of the node (assigned during add-node)
+                * ip - the ip of the node (assigned during add-node)
+                * status - UP (running), DOWN (stopped) or FAILED (was UP but exited with non-0 status since)
+                * return_code - the exit code of the Scylla process (only when status is FAILED)
+                * pid - the pid of the Scylla process
+                """),
+            help="status of the cluster")
+    parser_reload_config = subparsers.add_parser(
+            "reload-config",
+            formatter_class=RawDescriptionDefaultsHelpFormatter,
+            description=textwrap.dedent(f"""\
+                Reload the configuration from {CONFIG_FILE}
+
+                The configuration changes (if any) are not applied. The configuration changes,
+                will be picked up when nodes are added/started/restarted (depending on the
+                changed config items).
+                Be mindful of the fact that {CONFIG_FILE} is also written by the {app_name}
+                during create-cluster, add-node and remove-node.
+                """),
+            help=f"reloads the config from {CONFIG_FILE}, but changes are not applied, nodes will pick up changes when restarted")
+
+    return parser.parse_args(argv)
+
+
+class Supervisor:
+    def __init__(self, app_name):
+        logging.basicConfig(
+                filename=LOG_FILE,
+                filemode='a',
+                format='%(asctime)s,%(msecs)d %(name)s %(levelname)s %(message)s',
+                datefmt='%H:%M:%S',
+                level=logging.DEBUG)
+
+        self._logger = logging.getLogger('supervisor')
+
+        self._logger.info(f"Initializing supervisor (pid: {os.getpid()})")
+
+        self._app_name = app_name
+        self._config = {}
+        self._nodes = {}
+        self._error = None
+
+        with open(PID_FILE, "w") as f:
```

Comment:
probably should open the `PID_FILE` with mode `"x"`? and handle `FileExistsError`, in case the developer already has a cluster up and running managed by sscm?